model/strawberry/input/wikibase_input.py

```diff
@@ -0,0 +1,28 @@
+"""Wikibase - INPUT"""
+
+from typing import Optional
+import strawberry
+
+from model.enum.wikibase_category_enum import WikibaseCategory
+from model.strawberry.input.wikibase_url_input import WikibaseURLSetInput
+
+
+@strawberry.input
+class WikibaseInput:
+    """Wikibase"""
+
+    wikibase_name: str
+
+    organization: Optional[str] = None
+
+    description: str
```

Comment:
I think the description could be optional too, at least it is in the `WikibaseModel`.